packages/core/src/application_ref.ts

```diff
@@ -42,9 +42,8 @@ import {createEnvironmentInjector, NgModuleFactory as R3NgModuleFactory} from '.
 import {publishDefaultGlobalUtils as _publishDefaultGlobalUtils} from './render3/util/global_utils';
 import {TESTABILITY} from './testability/testability';
 import {isPromise} from './util/lang';
-import {scheduleMicroTask} from './util/microtask';
 import {stringify} from './util/stringify';
-import {NgZone, NoopNgZone} from './zone/ng_zone';
+import {IS_STABLE, isStableFactory, NgZone, NoopNgZone} from './zone/ng_zone';
```

Comment:
nit: we can probably be a bit more specific on what this token represents (currently the naming suggests it's kind of a flag :))
```suggestion
import {IS_STABLE_FN_IMPL, isStableFactory, NgZone, NoopNgZone} from './zone/ng_zone';
```